Cargo.toml

```diff
@@ -1,5 +1,6 @@
 [workspace]
 members = ["linker-diff", "linker-layout", "linker-trace", "linker-utils", "wild", "wild_lib"]
+default-members = ["wild"]
```

Comment:
Might be good to do this in a separate PR, since it's not related. Also, I was trying to understand why some of the changes you made in `args.rs` were able to compile and it turns out that they don't, but CI didn't pick it up because those changes weren't being built / tested on account of this change. I've changed the CI now so that even with this change, the build failure will show up. So after https://github.com/davidlattimore/wild/pull/200 - you should now have a build failure to fix next time you run CI - or you can do `cargo test --workspace` locally.

Reply:
Wow - I didn't see that one coming (obviously!)!!

I broken the golden rule when debugging the issue, undo all your changes vs. the version that works, even the ones you _think_ are unrelated :-(

Will fix this morning